launcher/ui/dialogs/CopyInstanceDialog.cpp

```diff
@@ -117,6 +122,40 @@ QString CopyInstanceDialog::instGroup() const
     return ui->groupBox->currentText();
 }
 
+const InstanceCopyPrefs& CopyInstanceDialog::getChosenOptions() const
+{
+    return m_selectedOptions;
+}
+
+void CopyInstanceDialog::checkAllCheckboxes(const bool& b)
+{
+    ui->keepPlaytimeCheckbox->setChecked(b);
+    ui->copySavesCheckbox->setChecked(b);
+    ui->copyGameOptionsCheckbox->setChecked(b);
+    ui->copyResPacksCheckbox->setChecked(b);
+    ui->copyShaderPacksCheckbox->setChecked(b);
+    ui->copyServersCheckbox->setChecked(b);
+    ui->copyModsCheckbox->setChecked(b);
+}
+
+// Sets b to true if state is a checked checkbox
+void CopyInstanceDialog::checkBool(bool& b, const int& state)
+{
+    if(state == Qt::Unchecked)
+    {
+        b = false;
+    }
+    else if(state == Qt::Checked)
+    {
+        b = true;
+    }
```

Comment:
```suggestion
    b = (state == Qt::Checked);
```

my change assumes `state` is not a tristate (at least from reading the function comment). Is that true?

Reply:
Yes, the checkbox is not tristate, although the previous implementation had used those two if statements as well so I kept them in case it meant anything, I could change it to that single line